packages/api-explorer/src/AuthBox.jsx

```diff
@@ -1,40 +1,34 @@
+/* eslint-disable no-restricted-syntax */
```

Comment:
Why this disable? If are rules not wanted, we could insert in `.eslintrc` file

Reply:
I dind't want to change eslint rules for the whole project but ok

Reply:
> *If are rules not wanted*, otherwise skip only the line not all the file

Reply:
I don't want that rules, I think they are uselessly strict, I've moved them :)